keyboards/rgbkb/pan/rules.mk

```diff
@@ -0,0 +1,51 @@
+# Processor choice= comment out either line 2 or 3.
+# ATmega32A = yes
```

Comment:
I would rather this be done with separate `32a` and `proton_c` revisions.

Reply:
If I were to do a new revision for the board overall, how would we handle that? Just have twice as many revision folders?

Reply:
Depends, are you planning to have 32A and STM32 versions of the new revision, or would it be just a single STM32 revision 2?

Reply:
If I were to add/change features on the board, the pinout for 32A and STM32 may change.

Reply:
OK, I've updated the PR